api/server.py

```diff
@@ -7,6 +7,43 @@ def run(port, hostname, debug_mode):
     app.run(host=hostname, port=port, debug=debug_mode)
 
 
-@app.route("/")
-def hello_world():
+# TODO: If meta-paths for A and B will be written in Java, they will need this information in Java
```

Comment:
@KDD-OpenSource/computation This point concerns you. If you'll write this functionality in Java, we have to pass it to you in another way (maybe over some Cypher query).

Reply:
@Baschdl maybe leave his for later?